src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/multiarch.py

```diff
@@ -166,35 +157,30 @@ def create_build(location,
         except FileNotFoundError as e:
             logger.warning("Unable to find file %s", str(e))
 
-def acr_multi_build_definition_create(registry_name,
-                                      repository,
-                                      tags=None,
+def acr_multi_build_definition_create(registry,
+                                      images=None,
```

Comment:
Hmmm images is also used in the YAML schema, so I thought it was a good idea to keep a standard between the two options.